File: app/src/main/java/tarena/com/coolcartoon/Shelf/SubscribeFragment.java
package tarena.com.coolcartoon.Shelf;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import tarena.com.coolcartoon.R;

/**
 * 订阅界面
 * Created by tarena on 2017/7/7.
 */

public class SubscribeFragment extends Fragment {
    View view;
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_subscribe,null);
        return view;
    }
}

File: app/src/main/java/tarena/com/coolcartoon/view/shelf_fragment/SubscribeFragment.kt
package tarena.com.coolcartoon.view.shelf_fragment


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import tarena.com.coolcartoon.R


/**
 * 订阅书籍
 */
class SubscribeFragment : Fragment() {


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater!!.inflate(R.layout.fragment_history, container, false)
    }

}// Required empty public constructor
